content/legal-notices/sub-processors/v6.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Sub-Processors
+lastUpdated: '2021-12-06T21:00:00.0Z'
+description: A list of sub-processors we use.
+---
+
+To support delivery of our services, Larder Software Limited, trading as Roadie, may engage and use data processors ("Sub-processors") with access to certain personal information.
```

Comment:
The casing of 'Sub-processors' in here and below is different than in the description.

Reply:
I won't fix this now.